<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CacheView</class>
 <widget class="QWidget" name="CacheView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="view">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>120</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="indicator">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string extracomment="Space used">%v/%m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="capacitySelector">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Remove Selected</string>
       </property>
       <property name="icon">
        <iconset resource="../res/images.qrc">
         <normaloff>:/control/bin16.png</normaloff>:/control/bin16.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>Clear All</string>
       </property>
       <property name="icon">
        <iconset resource="../res/images.qrc">
         <normaloff>:/control/broom16.png</normaloff>:/control/broom16.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRemoveSelected">
   <property name="icon">
    <iconset>
     <normaloff>:/control/error.png</normaloff>:/control/error.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="icon">
    <iconset>
     <normaloff>:/file/discard.png</normaloff>:/file/discard.png</iconset>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonRemove</sender>
   <signal>pressed()</signal>
   <receiver>actionRemoveSelected</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClear</sender>
   <signal>pressed()</signal>
   <receiver>actionRemoveAll</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
